src/analysis/retail/mage/shared/components/guide/CastDetails.tsx

```diff
@@ -0,0 +1,484 @@
+import { ReactNode, useState } from 'react';
+import { SpellIcon, TooltipElement } from 'interface';
+import Spell from 'common/SPELLS/Spell';
+import { QualitativePerformance } from 'parser/ui/QualitativePerformance';
+import { qualitativePerformanceToColor } from 'interface/guide';
+import styled from '@emotion/styled';
+
+const CastDetailsContainer = styled.div<{ performance: QualitativePerformance }>`
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  padding: 16px;
+  background: rgba(0, 0, 0, 0.3);
+  border-left: 4px solid ${(props) => qualitativePerformanceToColor(props.performance)};
+  border-radius: 4px;
+  margin-bottom: 12px;
+  transition: all 0.2s ease;
+
+  &:hover {
+    background: rgba(0, 0, 0, 0.4);
+    transform: translateX(2px);
```

Comment:
this `translate` is very distracting. typically, you'd use something like this to indicate an interactive element---not an entire container. the container itself isn't interactive (as far as I can tell), so the `&:hover` block should likely be cut entirely